main.py

```diff
@@ -54,25 +55,29 @@ def __init__(self, **kwargs):
             allowed_mentions=discord.AllowedMentions(everyone=False, roles=False),
             **kwargs,
         )
-        self.session = ClientSession(loop=self.loop)
```

Comment:
We'll have no typehint on this if we remove it completely, typehint here or as class annotation
This should really also be handled for other variables that don't comply

Reply:
#188